app/helpers/fhir_helper.rb

```diff
@@ -125,6 +126,15 @@ def patient_from_fhir(patient, default_jurisdiction_id)
     primary_phone = patient&.telecom&.find { |t| t&.system == 'phone' }
     secondary_phone = patient&.telecom&.select { |t| t&.system == 'phone' }&.second
     email = patient&.telecom&.find { |t| t&.system == 'email' }
+    # We want to allow the users to provide either the code or name of the Language
+    if patient&.communication&.first&.language&.coding&.first&.code.nil?
```

Comment:
Can you use something like dig here? https://apidock.com/ruby/Hash/dig

Reply:
No you cannot do this with a fhir patient (since it isnt a hash, just some class)